build/README.md

```diff
@@ -50,6 +50,10 @@ To run our external documentation tests:
 
 `cargo +nightly test --doc`
 
+To run our benchmarks:
+
+`cargo bench`
```

Comment:
```suggestion
`cargo bench`

We use [criterion](https://github.com/bheisler/criterion.rs) for benchmarking. You can find visual reports under `./target/criterion` for easy comparison.
```

^ small nit, just to make things easier for those not familiar with Rust bechmarking tools.

Reply:
Just gently bumping this comment, just in case you didn't see it.  Also updated the text, didn't like the "easy comparison" bit.